src/flashcart/flashcart.c

```diff
@@ -97,6 +98,7 @@ flashcart_err_t flashcart_init (void) {
 
         case CART_EDX:  // Series X EverDrive-64
```

Comment:
add `break` here as EDX flashcarts would use ED implementation and that wouldn't work at all

Reply:
Done.